get_notebook.py

```diff
@@ -0,0 +1,8 @@
+import os
+import galaxy
+import shutil
+
+hid = os.environ.get('DATASET_HID', None)
+if hid not in ('None', None):
+    galaxy.get(int(hid))
+    shutil.copy('/import/%s' % hid, '/import/ipython_galaxy_notebook.ipynb')
```

Comment:
`('None', None)` is because `$DATASET_HID` in the environment can be the literal string `"None"`. This could have been:

```python
hid = os.environ.get('DATASET_HID', 'None')
if hid == 'None':
```

But that somehow felt ickier.